pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -119,34 +121,66 @@ func (pc *PackageControllerClient) EnableCuratedPackages(ctx context.Context) er
 		noProxy := fmt.Sprintf("proxy.NO_PROXY=%s", strings.Join(pc.noProxy, "\\,"))
 		values = append(values, httpProxy, httpsProxy, noProxy)
 	}
-	if pc.eksaSecretAccessKey == "" || pc.eksaAccessKeyID == "" {
+	if (pc.eksaSecretAccessKey == "" || pc.eksaAccessKeyID == "") && pc.registryMirror == nil {
 		values = append(values, "cronjob.suspend=true")
 	}
 
-	err := pc.chartInstaller.InstallChart(ctx, pc.chart.Name, ociURI, pc.chart.Tag(), pc.kubeConfig, "", values)
+	writer, err := filewriter.NewWriter(pc.clusterName)
+	defer writer.CleanUpTemp()
```

Comment:
I would avoid doing this. When invoked from the a `cluster` workflow, it will delete al the filed in the `generated` folder. We only do this after a successful completion of the whole workflow.